src/Uno.UWP/UI/Composition/CoreAnimation.iOSmacOS.cs

```diff
@@ -227,7 +229,7 @@ private EventHandler<CAAnimationStateEventArgs> OnAnimationStopped(CAAnimation a
 		{
 			// This callback will be invoked when the animation is stopped, no matter the reason (completed, paused or canceled)
 
-			EventHandler<CAAnimationStateEventArgs> handler = default;
+			EventHandler<CAAnimationStateEventArgs>? handler = default;
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'handler'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932478)